server/src/main/resources/org/elasticsearch/bootstrap/test-framework.policy

```diff
@@ -40,6 +40,9 @@ grant codeBase "${codebase.lucene-test-framework}" {
   permission java.nio.file.LinkPermission "hard";
   // needed for RAMUsageTester
   permission java.lang.RuntimePermission "accessDeclaredMembers";
+  // needed for file permission tests to access user information
+  permission java.lang.RuntimePermission "accessUserInformation";
+  permission java.lang.RuntimePermission "getFileStoreAttributes";
```

Comment:
If I understand correctly, we had something that was working on Linux and macOS but problematic on Windows? And the fix for Windows is now problematic on Linux and macOS? Can we merely use different logic on Windows versus Linux/macOS and avoid this problem altogether. If I am not understanding correctly and that is not possible, I would like us to avoid adding these permissions for a single test. Instead, could we get away with an evil test that does not use the security manager at all? This should be a last resort, I would like to know if the if (Windows) else (Linux/macOS) approach would be acceptable first.